module/VuFind/src/VuFind/Controller/LibraryCardsController.php

```diff
@@ -78,10 +78,17 @@ public function homeAction()
         // Connect to the ILS for login drivers:
         $catalog = $this->getILS();
 
+        $config = $this->getConfig();
+        $allowConnectingCards = !empty(
+            $config->Catalog
+                ->auth_based_library_cards
+        ) &&
+            ($this->getAuthManager()->getAuthMethod() == 'Shibboleth');
```

Comment:
I don't think we want this hard-coded Shibboleth restriction anymore. Should we check for the callability of the connectUserCard method here, or just trust that the user configured things correctly and let the system fail elsewhere if there's a problem?